vast/integration/vast-transforms.yaml

```diff
@@ -19,6 +19,13 @@ vast:
       - hash:
           field: fileinfo.filename
           out: filename_hash
+    project_fields:
+      - projection:
```

Comment:
Up until now, all the transform names have been verbs which express the action the transform step takes. I think we should discuss renaming this to `project` rather than `projection` to keep the naming scheme intuitive.